google-drive/latest/support/index.md

```diff
@@ -8,6 +8,7 @@ The following are the supported platforms for Google Docs Integration:
 
 | Version | Notes |
 | ------- | ----- |
+| Content Services 7.3 | |
```

Comment:
I think we could simplify this list to just ACS 7.3 + ACS 7.2 + Community 7.3 (since the latest Community Edition will be based on ACS 7.3)
```suggestion
| Content Services 7.3 | |
| Content Services 7.2 | |
| Community Edition 7.3 | |
```